teamscale-client/src/main/java/com/teamscale/client/EReportFormat.java

```diff
@@ -3,6 +3,7 @@
 /** Enum of report formats. */
 public enum EReportFormat {
 	JACOCO("JaCoCo Coverage"),
+	JUNIT("JUnit Results"),
```

Comment:
Do these literals have to be in synch with the org.conqat.engine.index.shared.EReportFormat? If so, document in inline comment?